### ISSUE TITLE
Package management for every major platform

### ISSUE THREAD

**Issue by psmarcin** [g2a-com/klio#21]
### Current status
To install `klio` you need `go` and some basic knowledge about compiling. It's not very user friendly. 

### Idea
It would be awesome to have oneliner to install `klio` on every platform. 
Don't have much experience with windows and linux platforms but for macOS there is awesome tool called `brew`. 

### Implementation
Basically there are two ways to use `brew`, official - which requires some "popularity" for library and TAP (https://docs.brew.sh/Taps). As a temporary solution I think we can use tap. It works very well with `goreleaser` and github actions to make it fully automated.

**Comment by qzb** [comment]
Yep, it's a definitely a must have, but it doesn't have a priority right now, since klio is not yet usable for a general audience and we are not going to release any versions before 3.0.0.
